src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.Composite.sfxproj

```diff
@@ -32,6 +32,9 @@
 
   <ItemGroup>
     <PublishReadyToRunCrossgen2ExtraArgsList Include="--compositekeyfile:$(AssemblyOriginatorKeyFile)"/>
+    <!-- Compile with avx2 on x64 -->
+    <PublishReadyToRunCrossgen2ExtraArgsList Condition="'$(TargetArchitecture)' == 'x64'" Include="--inputbubble"/>
```

Comment:
currently `inputbubble` is required with `instruction-set`. Shouldnt `composite` just imply that? @AntonLapounov @trylek ?

Reply:
Yes, it implies; I think the check in Crossgen2 should be relaxed accordingly.  @davidwrighton has filed dotnet/sdk#17760, but I am not sure why it is filed on the SDK side.

Reply:
yeah this needs to be fixed in cg2. perhaps we should just move the bug over.